### PR TITLE
PT-13150 - Improve button rendering for Safari browser

### PR DESCRIPTION
**Pull request by swDennis**

### CODE REVIEW

File: Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.express-checkout-button.js [hunk 1]
```diff
@@ -277,7 +288,7 @@
          * @param {number} timeout
          * @return {number}
          */
-        buffer: function(fn, timeout) {
+        buffer: function (fn, timeout) {
```
Comment (mitelg) [inline Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.express-checkout-button.js:291]: is this method still in use?

File: Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.in-context-checkout.js [hunk 2]
```diff
@@ -69,28 +70,36 @@
...
                     }
                 });
-            } else {
-                this.paypal = window.paypal;
-                this.renderButton();
             }
         },
 
-        renderSdkUrl: function() {
+        payPalObjectCheck: function () {
```
Comment (mitelg) [inline Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.in-context-checkout.js:92]: is it possible to abstract this method? as it is used in every plugin

File: plugin.xml [hunk 3]
```diff
@@ -5,13 +5,22 @@
...
+    <version>6.1.5</version>
     <copyright>(c) by shopware AG</copyright>
     <license>MIT</license>
     <link>http://store.shopware.com</link>
     <author>shopware AG</author>
     <compatibility minVersion="5.2.27" maxVersion="5.99.99"/>
 
+    <changelog version="6.1.5">
+        <changes lang="de">
+            PT-13146 - Behebt ein Problem mit dem Anzeigen der PayPal Buttons im Safari Browser;
```
Comment (mitelg) [inline plugin.xml:17]:
```suggestion
            PT-13146 - Behebt ein Problem mit der Anzeige der PayPal Buttons im Safari Browser;
```

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.express-checkout-button.js (hunk 1 above)
Comment (swDennis) [reply Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.express-checkout-button.js:291]: I remove that

File: Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.in-context-checkout.js (hunk 2 above)
Comment (swDennis) [reply Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.in-context-checkout.js:92]: like discussed

**Review by mitelg — APPROVED** [review]

**Pull request merged**